### DISCUSSION TITLE
0.9m03 - Text Pages 1

### DISCUSSION THREAD

**Discussion by maple7-7-7** [trufanov-nok/minidjvu-mod#8] [General]
Hi Alex!
You have finally beat the system!
I compared two DjVus, one made using your minidjvu-mod09.3, and the other using DjVuSolo3.1. Both programs used the same input files: the 10 pbms as pages 1-10.
Both used the 10-page dictionary function. Both were set to 500 dpi, and both were encoded as bitonal.
The result: Your DjVu is _smaller_ than theirs!
29820 bytes vs 30,092 bytes
DjVuSolo added 55 characters to the 11 files' page names, as a "_0001" suffix, but even allowing for this, if we look at the individual DjVu page file sizes, we get the following:
29667 bytes (yours) vs 29934 bytes (theirs)
Their only edge is in dictionary file size:
3530 bytes (yours) vs 3479 bytes (theirs)
See image, with Solo on the left and minidjvu-mod on the right.
I think we can now say with even more confidence, that your program is essentially equivalent to the commercial version in bitonal compression.
This is great. Congratulations!

![DjVuSolo_vs_minidjvu-mod__10_pbm_pgs](https://user-images.githubusercontent.com/58869774/128745702-f9716b73-990d-4ee5-8766-10cf70c79461.png)

**Comment by maple7-7-7** [comment]
Hi Alex,
Your minidjvu-mod is pretty much now my go-to DjVu encoder for standard text pages of the sort we have been testing. Thanks! I have also been testing your program in the case of Tables and find that the resulting DjVus from PBMs are about 60% larger in file size than those from the any2djvu server or DjVuSolo. Is it possible to improve on this, or is it largely irrelevant to your project? The chief new factors are line art, the use of many digits, and superscripts. Thanks. (And did you check my previous post?)

**Comment by trufanov-nok** [comment]
Hi,
I'm still working on accuracy improvement and bold/normal detection. 
As for tables: could you attach source images, as well as minidjvu-mod encoded and any2djvu server encoded djvus?

**Comment by trufanov-nok** [comment]
Also make sure that you are using the latest minidjvu-mod build. It has some improvements in long black lines encoding
